lib/utilities/global-npm-version.js

```diff
@@ -0,0 +1,9 @@
+// Get the global npm version. You cannot require global npm modules, including
+// global npm itself, but you can still *run* global npm.
+const spawnSync = require('child_process').spawnSync;
```

Comment:
We can't use `const` or `spawnSync` on platforms older than 0.12 (we support 0.10 still).